src/components/AssetScanner/Webcam/Webcam.tsx

```diff
@@ -170,7 +167,7 @@ export class Webcam extends Component<WebcamProps, WebcamState> {
     return (
       <StyledVideo
         autoPlay={true}
-        muted={this.props.audio}
+        muted
```

Comment:
I don't know the gearbox, but it looks like the previous solution had some mechanism to turn on/off the audio. Should we always disable audio?

Reply:
Does look like that but I cant think if why we would need the audio on the Webcam component that is only used y the WebCamScanner (and only by Infield)

Reply:
@YevheniiMaltsev the underlying problem is that the component should have been muted based on the props.audio passed but the condition was wrong ...  I've changed the condition now, please check again.

Reply:
Yeah, sorry for confuse. I didn't request changes, I just wanted to understand if it's okay 😊